### DISCUSSION TITLE
Diarization result for "End-to-end speaker segmentation for overlap-aware resegmentation"

### DISCUSSION THREAD

**Discussion by ChokJohn** [pyannote/pyannote-audio#795] [Q&A] [answered]
I use the pre-trained model directly for the speaker diarization task. But I got a bad result on AMI dataset, is this correct ？
AMI Mix-Headset ：
FA = 0.06082287973986739 
Miss. =  0.08262689808073657
Conf.  = 0.46741261779675336

**Comment by hbredin** [comment, marked as answer]
Yes, that is probably correct.

The sum of `FA` and `Miss.` does match the numbers reported in the paper.
The high `Conf.` is due to the fact that the segmentation model is not capable of tracking speakers over time (it only works on small 5s chunks). 

You'd have to combine the segmentation model with speaker embedding to perform proper diarization.
See this [paper](https://arxiv.org/abs/2109.06483) and [code](https://github.com/juanmc2005/StreamingSpeakerDiarization/) for a way to do that.

**Reply by 385614027** [reply]
你好，我想问下，例如，HIT,MISS  MH,FA他们是怎样计算的，根据的是什么？？？

**Comment by Yagna24** [comment]
Hi @ChokJohn , 
How did you fetch these results, how to perform evaluation here ?
